src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -58,6 +58,13 @@
    :dependencies (schema/pred sequential?)
    :fns-map FnsMap})
 
+(def ProtocolandDependenciesMap
+  "Schema defining the map used to represent the protocol and dependencies
+  of a service."
+  {:fns [clojure.lang.PersistentList]
```

Comment:
@nathanielksmith ping.  Would like to get this merged; I have a slight pref for changing it to something more clojure-y but would be OK with this version if there's something you thought of that would warrant it.